### PR TITLE
Minor cleanups in `oracle::Actor`

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: daemon/src/oracle.rs [hunk 1]
```diff
@@ -16,22 +16,14 @@ use std::collections::{HashMap, HashSet};
 const OLIVIA_EVENT_TIME_FORMAT: &[FormatItem] =
     format_description!("[year]-[month]-[day]T[hour]:[minute]:[second]");
 
-pub struct Actor<CFD, M>
-where
-    CFD: xtra::Handler<Attestation>,
-    M: xtra::Handler<Attestation>,
```
Comment (luckysori) [inline daemon/src/oracle.rs:22]: :+1: I'm surprised to see that we don't need these trait bounds everywhere. I swear we tried to elide them before and it didn't work.

**Review by luckysori — APPROVED** [review]

File: daemon/src/oracle.rs (hunk 1 above)
Comment (thomaseizinger) [reply daemon/src/oracle.rs:22]: I moved the methods into dedicated impl blocks.

Comment (luckysori) [reply daemon/src/oracle.rs:22]: We also tried that, but we clearly did something wrong.

**Review by da-kami — APPROVED** [review]

**Pull request merged**